.eslintrc

```diff
@@ -17,6 +17,9 @@
     },
     {
       "files": ["**/*.tsx"],
+      "globals": {
+        "JSX": true
```

Comment:
why is this a global?

Reply:
It's use to do this:
```tsx
const Components = ():JSX.Element =>{...}
```

Reply:
but that's a TS type, not a runtime value

Reply:
Exactly as @ljharb you shouldn't be changing this file. If something needs to be changed is the tsconfig to have "jsx" as "detect" or "next".

Reply:
True, the ts type has been identified as undefined by eslint

Reply:
are you using the TS parser?

Reply:
@ljharb I suppose we are. (https://github.com/vercel/next.js/blob/canary/packages/eslint-config-next/index.js#L93) but we do not have the JSX types imported. (It might be that React is not exposing the JSX namespace anymore)